pdns/dnsdistdist/configure.ac

```diff
@@ -74,7 +74,7 @@ PDNS_ENABLE_SANITIZERS
 
 PDNS_CHECK_PANDOC
 
-LDFLAGS="$RELRO_LDFLAGS $LDFLAGS"
+LDFLAGS="$RELRO_LDFLAGS $LDFLAGS -lfstrm"
```

Comment:
The configure needs to be detect whether libfstrm is available, and dnsdist should be able to compile correctly if it isn't.